### DISCUSSION TITLE
Can I homehow disable login?

### DISCUSSION THREAD

**Discussion by naddel91** [LizardByte/community#429] [Sunshine]
I would like to start sunshine immediately on boot without login.

**Comment by ReenigneArcher** [comment]
Just install Sunshine using our installer and make sure the service is selected.

**Comment by naddel91** [comment]
Launch on startup is installed by default, but that does not bypass the login nag screen.

**Reply by ReenigneArcher** [reply]
I would highly advice NOT to disable login, but https://www.wintips.org/disable-login-screen-in-windows-10-11/

**Comment by naddel91** [comment]
No I mean sunshines login at startup, not windows-login.

**Reply by ReenigneArcher** [reply]
Sunshine's only login is for the WebUI, that can't be disabled. Any other login prompt you get is from your OS.

[a comment by an automated account is not shown]
